data-source/jsonnet/england-wales/household/blocks/who-lives-here/any_visitors.jsonnet

```diff
@@ -18,6 +18,7 @@ function(census_date) {
     id: 'any-visitors-question',
     title: questionTitle(census_date),
     mandatory: true,
+    hide_on_section_summary: true,
```

Comment:
`Does anyone usually live at` question need to also be hidden from the summary. Also these need to be hidden on all region schemas.

<img width="687" alt="Screenshot 2019-09-03 at 09 12 11" src="https://user-images.githubusercontent.com/35296336/64155565-ee17db80-ce2a-11e9-9f1e-b893ebe2ddad.png">